### ISSUE TITLE
Questions block

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/page-constructor#488]
1. Each question element should be accessible via TAB navigation and expandable on Enter
1. Each question element should have `aria-expanded` attribute set correctly
